wsd/ClientRequestDispatcher.cpp

```diff
@@ -763,6 +764,10 @@ void ClientRequestDispatcher::handleIncomingMessage(SocketDisposition& dispositi
 
                     launchAsyncCheckFileInfo(_id, accessDetails, RequestVettingStations);
                 }
+                if (request.getMethod() == Poco::Net::HTTPRequest::HTTP_POST)
```

Comment:
Would be good to have a comment explaining what's going on here. This is completely non-intuitive and we spent quite a bit of time to figure this workaround out. I know I'll likely forget what this is doing in a few short weeks.